### PR TITLE
Feature/adf 89/tabular import integration branch

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/ADF-89

### CODE REVIEW

File: templates/import/item_match_correct_response.xml.tpl [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                <choiceInteraction responseIdentifier="RESPONSE" shuffle="<?=get_data('shuffle')?>" maxChoices="<?=get_data('maxChoices')?>" minChoices="<?=get_data('minChoices')?>" orientation="vertical">
+                    <prompt><?=get_data('question')?></prompt>
+                    <?foreach(get_data('choices') as $choice):?>
+                        <simpleChoice identifier="<?=$choice->getId()?>" fixed="false" showHide="show"><?=_dh($choice->getChoice())?></simpleChoice>
+                    <?endforeach?>
+                </choiceInteraction>
+            </div>
+        </div>
+    </itemBody>
+    <responseProcessing>
```
Comment (ktsirkou) [inline templates/import/item_match_correct_response.xml.tpl:43]: Why not use the template directly http://www.imsglobal.org/question/qtiv2p2/rptemplates/match_correct.xml
Also the outcome variable SCORE_RESPONSE seems redundant.

**Review by gabrielfs7 — CHANGES_REQUESTED** [review]
We cannot forget to

- [x] Fix all broken tests
- [ ] Update composer.json if required 
- [ ] Cover all TODOs in the list if are blocker

**Comment by AntonNeviarouski** [comment]
QA verified on http://tabular-adf-156-items-import.playground.kitchen.it.taocloud.org:46001/

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**